scripts/webpack/webpack.config.ts

```diff
@@ -98,6 +99,10 @@ const webpackConfig: webpack.Configuration = {
         reactVis: require('react-vis/package.json').version,
       },
     }),
+    new ResourcePreloadWebpackPlugin({
```

Comment:
Can you add a comment about why this was needed?

Reply:
Sure, I am also thinking to run this only for Screener builds